sequencer/src/api.rs

```diff
@@ -348,10 +348,20 @@ pub mod test_helpers {
             builder_port: Option<u16>,
         ) -> Self {
             let mut cfg = TestConfig::default_with_l1(l1);
+            cfg.builder_port = builder_port;
 
-            let (builder_task, builder_url) = run_test_builder(builder_port).await;
+            Self::with_state_config(opt, state, persistence, catchup, cfg).await
+        }
 
-            cfg.set_builder_url(builder_url);
+        pub async fn with_state_config(
+            opt: Options,
```

Comment:
I think variable names like `opt` and `config` are not very revealing. I believe these options are for initializing sequencer api? Can we use a variable name that help us to understand that?

Reply:
`opt` and `cfg` are commonly used in the relevent code. I just kept them...

Reply:
The confusion arises from having a single function that takes both an `opt` and a `cfg`, which I believe was introduced in this PR.

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/1583/commits/b6b3d62292db11b4120d54866d6f834304797cd6
Renamed `opt` to `api_config` and `config` to `network_config`